e2e/_suites/fleet/stand-alone.go

```diff
@@ -26,9 +26,10 @@ import (
 
 // StandAloneTestSuite represents the scenarios for Stand-alone-mode
 type StandAloneTestSuite struct {
-	Cleanup  bool
-	Hostname string
-	Image    string
+	Cleanup     bool
```

Comment:
Once merged this change, I'm super tempted to create an issue to merge both test suite structures.